trace/ftrace/tracer/ftrace-tracer.sh

```diff
@@ -69,7 +69,7 @@ tracername="function_graph"
 if grep -qw ${tracername} ${FTRACE_PREFIX}/available_tracers; then
   starttrace ${tracername}
   sleep 5
-  cat ${TRACE} &> /dev/null ; rtrn=$?
+  cat ${TRACE} | head &> /dev/null ; rtrn=$?
```

Comment:
Wait, this would actually be better as `head ${TRACE} &>/dev/null`. No need to use `cat` here if you're going to use `head` anyway.

Reply:
I'll try later on. I use cat previously to reduce the chance of infinite waiting (but it did not help enough). I'll see if head can do enough trick or not then follow-up with email or comment.